manifest/list_test.go

```diff
@@ -0,0 +1,151 @@
+package manifest
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/containers/image/types"
+	digest "github.com/opencontainers/go-digest"
+	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func isOCI1Index(i interface{}) bool {
+	switch i.(type) {
+	case *OCI1Index:
+		return true
+	}
+	return false
+}
+
+func isSchema2List(i interface{}) bool {
+	switch i.(type) {
+	case *Schema2List:
+		return true
+	}
+	return false
+}
+
+func cloneOCI1Index(i interface{}) ManifestList {
+	if impl, ok := i.(*OCI1Index); ok {
+		return OCI1IndexClone(impl)
+	}
+	return nil
+}
+
+func cloneSchema2List(i interface{}) ManifestList {
```

Comment:
The four helper methods above are no longer used.

Reply:
Still outstanding.

Reply:
Okay, they should be gone now.